### ISSUE TITLE
Releasing a new gem version

### ISSUE THREAD

**Issue by brendon** [swanandp/acts_as_list#196]
@fabn, could you run me through how the new change log generator works? I'd like to release a new gem version.

**Comment by swanandp** [comment]
👍

**Comment by fabn** [comment]
Hi @brendon detailed instructions are available [https://github.com/skywinder/github-changelog-generator] 

### TL;DR

* Generate a github token [here](https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token) for your github username
* Release the gem and push the tag to github
* After you pushed to github (changelog needs the tag available online) run the rake task
* Inspect and commit updated changelog
* Push it :dancers: :dancers: 

```bash
rake release
git tag vXYZ
git push --tags
# After you get the token
export CHANGELOG_GITHUB_TOKEN="«your-40-digit-github-token»"
rake changelog
git diff # To check the changelog
git -a commit -m 'Updated changelog'
git push
```

**Comment by swanandp** [comment]
Let's put this in a doc somewhere. Great job doing this @fabn!

**Comment by brendon** [comment]
Indeed :) Thanks @fabn. @swanandp, I'll have a crack at the instructions then post them in the wiki once I've confirmed they're working well :)

**Comment by brendon** [comment]
Hi guys, I've released the 0.7.3 gem version. It looks like `rake release` creates the tag in github also, so I didn't run `git tag...` and `git push --tags`.

The wiki page is here: https://github.com/swanandp/acts_as_list/wiki/Creating-a-new-gem-version

Let me know if there are any problems.

**Comment by swanandp** [comment]
Nice! :fireworks:
